administration/logging.md

```diff
@@ -61,7 +65,7 @@ The config file for logging is `org.ops4j.pax.logging.cfg` located in the `userd
 
 ## Defining what to log
 
-In order to see the messages, logging needs to activated defining what should be logged and in which detail. This can be done in Karaf using the following console command:
+In order to see the messages, logging needs to activated, defining what should be logged and in which detail. This can be done in Karaf using the following console command:
```

Comment:
Please split up to one line per Sentence.

Reply:
Done 👍